tests/functional/adapter/utils/test_utils.py

```diff
@@ -45,6 +65,11 @@ class TestConcat(BaseConcat):
     pass
 
 
+# Use either BaseCurrentTimestampAware or BaseCurrentTimestampNaive but not both
```

Comment:
some more explanation would be nice to have here since it's no clear why

Reply:
I think before dbt didn't distinguish between a time zone aware timestamp and a naive timestamp while we always worked with timezone aware (UTC) timestamps.